pushsnapscript/src/pushsnapscript/snap_store.py

```diff
@@ -5,11 +5,7 @@
 from mozilla_version.version import VersionType
 from scriptworker.exceptions import TaskVerificationError
 from scriptworker.utils import get_hash, get_single_item_from_sequence
-
-# XXX Hack to only import a subset of snapcraft. Otherwise snapcraft can't be built on any other
```

Comment:
This comment was making `black` and `isort` to output different results. It turns out it's not a valid comment anymore. That's why this change is here.